junebug/tests/helpers.py

```diff
@@ -257,6 +257,22 @@ def assert_body_contains(self, req, **fields):
             dict((k, v) for k, v in body.iteritems() if k in fields),
             fields)
 
+    def generate_status(
+            self, level=None, components={}, inbound_message_rate=0,
+            outbound_message_rate=0):
+        '''Generates a status that the http API would respond with, given the
+        same parameters'''
+        return {
+            'status': level,
+            'components': components,
+            'inbound_message_rate': inbound_message_rate,
+            'outbound_message_rate': outbound_message_rate,
+        }
```

Comment:
My thinking here is that when we add more fields, or change how the fields are organised, we can just change this function (with relevant defaults), instead of changing all of the relevant tests.